src/docs/transactions.dox

```diff
@@ -209,6 +209,17 @@ durable timestamp, instead of commit timestamp for persisting the data updates.
 If the durable timestamp is not specified, then the commit timestamp will be
 considered as the durable timestamp.
 
+There are a number of constraints around assigning timestamps for running
+transactions - the table below summarizes those constraints:
+
+| API     | Prepared | Constraint | Enforced | Description |
+|---------|----------|------------|----------|-------------|
+| Prepare | During | prepare_timestamp >= stable_timestamp | Y | None |
+| Commit  | No | commit_timestamp > stable_timestamp | Y | None |
+| Commit  | Yes | commit_timestamp >= prepare_timestamp | Y | The commit timestamp may be older than the oldest timestamp and the time of commit. |
```

Comment:
`and` I think should be `at`? It would be clearer with this tiny wording change: `The commit timestamp may be older than the oldest timestamp at the time of the commit API call.`